kod/include/blakston.khd

```diff
@@ -914,6 +914,15 @@
    RID_OLD_MARION = 9001
    RID_OLD_BAR_NORTH = 9002
    RID_OLD_BAR_SOUTH = 9003
+   
+   RID_ELEMENTAL_DUNGEON = 9005
```

Comment:
These RIDs shouldn't be in the historical section, but rather in their own section. The earlier the RID the better, but consider what RID block comes before and after it and possible expansion of each. The idea is to fill up the available space in a logical manner, making it clear which block is next to be filled, and what the likely number is for any given room.